src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -164,6 +166,8 @@ export class MeetingManager implements AudioVideoObserver {
     if (videoDownlinkBandwidthPolicy) {
       this.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
     }
+
+    this.eventDidReceiveObserver = { eventDidReceive: this.eventDidReceive };
```

Comment:
Is the code from L243 - L245, the `eventDidReceive` function used somewhere else? Could we do not declare this function and just add it into the `eventDidReceiveObserver` at L140?

Feel like we don't need L160-170 and L243 - L245.

Reply:
Yup updated the code to do it, I refrained as the `eventDidReceive` was an public arrow function so dint thought to remove it, but have removed as it should not be used publicly anyhow. Thanks!